package.json

```diff
@@ -54,22 +54,22 @@
     "url": "https://github.com/ExpediaGroup/service-client/issues"
   },
   "engines": {
-    "node": ">=8.0.0",
+    "node": ">=12.0.0",
     "npm": ">=5.0.0"
   },
   "dependencies": {
-    "@hapi/hoek": "^8.5.0",
-    "@hapi/joi": "^16.1.7",
-    "@hapi/wreck": "^16.0.1",
+    "@hapi/hoek": "^9.0.4",
+    "@hapi/wreck": "^17.0.0",
+    "joi": "^17.2.0",
```

Comment:
Would you please move `joi` to the end so that it's alphabetized?

Reply:
done